airsonic-main/src/main/java/org/airsonic/player/io/RangeOutputStream.java

```diff
@@ -55,7 +55,7 @@ public RangeOutputStream(OutputStream out, HttpRange range) {
      * @return The possibly wrapped output stream.
      */
     public static OutputStream wrap(OutputStream out, HttpRange range) {
-        if (range == null) {
+        if (range == null || (range.getFirstBytePos() == 0 && !range.isClosed())) {
```

Comment:
Hmm, can you provide any additional context for this change?

Reply:
Reading this again, and it makes sense although imo should be unnecessary assuming RangeOutputStream behaves correctly.

Reply:
Since it's not super-duper-obvious, a comment would be nice ♥

Reply:
Yeah, I wanted to short-circuit wrapping the stream if it's unnecessary. Added a comment to that effect.